dev/devicelab/bin/tasks/plugin_dependencies_test.dart

```diff
@@ -74,10 +75,6 @@ Future<void> main() async {
         );
       });
 
-      // https://github.com/flutter/flutter/issues/46898
-      // https://github.com/flutter/flutter/issues/39657
-      File(path.join(pluginCDirectory.path, 'android', 'build.gradle')).deleteSync();
```

Comment:
Removed this line since the top issue is closed and the second one I assume would be fixed by this PR.